clang-tools-extra/clang-change-namespace/ChangeNamespace.h

```diff
@@ -109,6 +110,9 @@ class ChangeNamespaceTool : public ast_matchers::MatchFinder::MatchCallback {
   };
 
   std::string FallbackStyle;
+  // Specifies the list of paths to be searched when FormatStyle or a BasedOnStyle
+  // in a .clang-format file specifies an arbitrary file to include
```

Comment:
```suggestion
  // in a .clang-format file specifies an arbitrary file to include.
```
Comments ending in full stop.